tests/test_schema_and_aiohttp.py

```diff
@@ -0,0 +1,410 @@
+)
+
+
+@pytest.fixture(scope="session")
+def schema() -> strawberry.Schema:
+    return create_schema(False)
+
+
+@pytest.fixture()
+async def client(aiohttp_client) -> TestClient:
+    cors = True
+    debug = False
+    graphiql = False
+    client = await aiohttp_client(create_app(cors, debug, graphiql))
+    return client
+
+
+async def query_aio_httpclient(client: TestClient, query: str):
+    resp = await client.get("/ws", params={"query": query})
+    result = await resp.json()
+    return result["data"]
+
+
+async def post_aio_httpclient(client: TestClient, query: str):
+    resp = await client.post("/ws", json={"query": query})
+    result = await resp.json()
+    return result["data"]
+
+
+async def query_schema(schema: strawberry.Schema, query: str):
+    result = await schema.execute(query)
+    return result.data
+
+
+@pytest.fixture()
+async def client_get_int_pv(client: TestClient, int_query: str):
+    return await query_aio_httpclient(client, int_query)
+
+
+@pytest.fixture()
+async def schema_get_int_pv(schema: strawberry.Schema, int_query: str):
+    return await query_schema(schema, int_query)
+
+
+@pytest.fixture()
+async def client_get_str_pv(client: TestClient, str_query: str):
+    return await query_aio_httpclient(client, str_query)
+
+
+@pytest.fixture()
+async def schema_get_str_pv(schema: strawberry.Schema, str_query: str):
+    return await query_schema(schema, str_query)
+
+
+@pytest.fixture()
+async def client_get_nan_pv(client: TestClient, nan_query: str):
+    return await query_aio_httpclient(client, nan_query)
+
+
+@pytest.fixture()
+async def schema_get_nan_pv(schema: strawberry.Schema, nan_query: str):
+    return await query_schema(schema, nan_query)
+
+
+@pytest.fixture()
+async def client_get_enum_pv(client: TestClient, enum_query: str):
+    return await query_aio_httpclient(client, enum_query)
+
+
+@pytest.fixture()
+async def schema_get_enum_pv(schema: strawberry.Schema, enum_query: str):
+    return await query_schema(schema, enum_query)
+
+
+@pytest.fixture()
+async def client_put_long_and_enum_pv(client: TestClient, long_and_enum_put: str):
+    return await post_aio_httpclient(client, long_and_enum_put)
+
+
+@pytest.fixture()
+async def schema_put_long_and_enum_pv(
+    schema: strawberry.Schema, long_and_enum_put: str
+):
+    return await query_schema(schema, long_and_enum_put)
+
+
+@pytest.fixture()
+async def client_put_list_pv(client: TestClient, list_put: str):
+    return await post_aio_httpclient(client, list_put)
+
+
+@pytest.fixture()
+async def schema_put_list_pv(schema: strawberry.Schema, list_put: str):
+    return await query_schema(schema, list_put)
+
+
+@pytest.fixture()
+async def client_put_base64_pv(client: TestClient, base64_put: str):
+    return await post_aio_httpclient(client, base64_put)
+
+
+@pytest.fixture()
+async def schema_put_base64_pv(schema: strawberry.Schema, base64_put: str):
+    return await query_schema(schema, base64_put)
+
+
+int_pv_result = dict(
```

Comment:
I'd find these easier to read if you used `{}` notation instead of `dict`; is there a reason for this choice?

Reply:
I was just following what was there before but I do agree that the `{}` notation is easier to read and also a more efficient way of creating dictionaries. I have updated all dictionary definitions in the tests.

Reply:
Probably because I don't like typing `"`, so I tend to write single level dicts as `dict(a=1, b=2, c=3)` rather than `{"a": 1, "b": 2, "c": 3}`. Agree that nested dicts are definitely cleaner with `{}` though.